docs/design/proposals/configure-snapshot-based-rbd-mirroring.md

```diff
@@ -0,0 +1,324 @@
+# Configure Snapshot-based RBD Mirroring
+
+---
+RBD mirroring is an asynchronous replication of RBD images between multiple
+Ceph clusters. The following guide demonstrates how to perform the basic
+administrative tasks to configure
+[snapshot-based mirroring](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/#create-image-mirror-snapshots)
+using the rbd command and handle Disaster Recovery scenarios.
+
+For more details about RBD Mirroring, refer to the official
+[Ceph documentation](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/).
+
+## Requirements
+
+* [Rook](https://rook.io/docs/rook/master/) should be installed
```

Comment:
the 1st link can point to rook.io, as you have a next link that points to the docs